wifi: nRF54 series doc updates (#15712)
nrfconnect/sdk-nrf: doc/nrf/device_guides/working_with_nrf/nrf70/nrf7002eb_54h20dk_gs.rst

:orphan:

.. _ug_nrf7002eb_nrf54h20dk_gs:

Getting started with nRF7002 EB for nRF54H20DK
##############################################

.. contents::
   :local:
   :depth: 4

This section builds on :ref:`nRF7002 EB <ug_nrf7002eb_gs>` documentation, only changes specific to :ref:`zephyr:nrf54h20dk_nrf54h20` are mentioned here.

Overview
********

The nRF7002 EB is a shield that can be used with the nRF54H20DK to add support for Wi-Fi.

Limitations
===========

The Wi-Fi support is experimental and has the following limitations:

- STA mode only
- Wi-Fi open security only
- Low throughput applications only

> Pin mapping for nRF54H20DK
> ==========================

Reviewer: ```suggestion
Pin mapping for nRF54H20 DK
===========================
```

> The pin mapping for the nRF54H20DK is as follows:

Reviewer: ```suggestion
The pin mapping for the nRF54H20 DK is as follows:
```

+------------------+-----------------------+-----------------------------------------------+

> | nRF700x pin name | nRF54H20DK pins       |  Function                                     |

Reviewer: ```suggestion
| nRF700x pin name | nRF54H20 DK pins       |  Function                                     |
```

| IOVDDEN          | P1.00                 | Enable IOVDD regulator                        |
+------------------+-----------------------+-----------------------------------------------+
| IRQ              | P1.02                 | Interrupt from nRF7002                        |
+------------------+-----------------------+-----------------------------------------------+
| GRANT            | P1.03                 | Coexistence grant from nRF7002                |
+------------------+-----------------------+-----------------------------------------------+
| REQ              | P1.08                 | Coexistence request to nRF7002                |
+------------------+-----------------------+-----------------------------------------------+
| STATUS           | P1.09                 | Coexistence status from nRF7002               |
+------------------+-----------------------+-----------------------------------------------+

Minimum requirements
********************

Make sure you have all the required hardware and that your computer and mobile device both have one of the supported operating systems.

Hardware
========

> * nRF54H20DK

Reviewer: ```suggestion
* nRF54H20 DK
```

:orphan:

.. _ug_nrf7002eb_nrf54h20dk_gs:

Getting started with nRF7002 EB for nRF54H20 DK
############################################### 

.. contents::
   :local:
   :depth: 4

> This section provides information about how to use nRF7002 :term:`Expansion Board (EB)` (PCA63561) to provide Wi-Fi® connectivity to the :ref:`zephyr:nrf54h20dk_nrf54h20`.

Reviewer 2: ```suggestion
This section provides information about how to use the nRF7002 :term:`Expansion Board (EB)` (PCA63561) to provide Wi-Fi® connectivity to the :ref:`zephyr:nrf54h20dk_nrf54h20`.
```

The changes mentioned are specific to the nRF54H20 DK.

> See the :ref:`nRF7002 EB <ug_nrf7002eb_gs>` documentation for more details on building and programming with nRF7002 EB.

Reviewer 2: ```suggestion
See the :ref:`nRF7002 EB <ug_nrf7002eb_gs>` documentation for more details on building and programming with the nRF7002 EB.
```

Overview
********

The nRF7002 EB is a shield that can be used with the nRF54H20 DK to add support for Wi-Fi.

Limitations
===========

The Wi-Fi support is experimental and has the following limitations:

* It only supports STA mode.
* It only allows Wi-Fi open security.

> * It is only suitable for low throughput applications.

Reviewer 2: ```suggestion
* It is only suitable for low-throughput applications.
```

Pin mapping for nRF54H20 DK
===========================

The pin mapping for the nRF54H20 DK is as follows:

+------------------+-----------------------+-----------------------------------------------+

> | nRF700x pin name | nRF54H20 DK pins       |  Function                                     |

Reviewer 2: ```suggestion
| nRF70 Series pin name | nRF54H20 DK pins       |  Function                                     |
```

| IOVDDEN          | P1.00                 | Enable IOVDD regulator                        |
+------------------+-----------------------+-----------------------------------------------+
| IRQ              | P1.02                 | Interrupt from nRF7002                        |
+------------------+-----------------------+-----------------------------------------------+
| GRANT            | P1.03                 | Coexistence grant from nRF7002                |
+------------------+-----------------------+-----------------------------------------------+
| REQ              | P1.08                 | Coexistence request to nRF7002                |
+------------------+-----------------------+-----------------------------------------------+
| STATUS           | P1.09                 | Coexistence status from nRF7002               |
+------------------+-----------------------+-----------------------------------------------+

Minimum requirements
********************

Make sure you have all the required hardware and that your computer and mobile device both have one of the supported operating systems.

> Hardware
> ========
> 
> * nRF54H20 DK
> * nRF7002 EB
> * A suitable battery
> * Micro-USB 2.0 cable
> * USB-C charger
> * Jumper wires

Reviewer 2: ```suggestion
Hardware
========

You must have the following hardware: 

* nRF54H20 DK
* nRF7002 EB
* A suitable battery
* Micro-USB 2.0 cable
* USB-C charger
* Jumper wires
```

Reviewer 2: Regarding Jumper wires, how many (minimum) are necessary? That should be put in, something like,

``Jumper wires - one for each <text here>``, or whatever the minimum is

Reviewer: I suggest not to include any text after Hardware to avoid redundancy, as its already there after Minimum Requirements sub-heading. See https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/gsg_guides.html.

I would suggest to keep the Jumper wires generic for now.
@krish2718: Will yo add the details after RC1?

Reviewer: Yes, that's why I suggested not to include "You must have the following hardware: " text after **Hardware**.
